app/storage/metadata_parser.py

```diff
@@ -97,43 +97,39 @@ class RunnerMetadataSchema(Schema, StripWhitespaceMixin):
     survey = VALIDATORS["string"](
         required=False, validate=validate.OneOf(("CENSUS", "CCS")), missing="CENSUS"
     )  # type:ignore
-    form_type = VALIDATORS["string"](
-        required=False, validate=validate.OneOf(("H", "I", "C"))
-    )  # type:ignore
     region_code = VALIDATORS["string"](
         required=False, validate=RegionCode()
     )  # type:ignore
 
+    # The following two parameters are for business schemas
+    form_type = VALIDATORS["string"](required=False)  # type:ignore
+    eq_id = VALIDATORS["string"](required=False)  # type:ignore
+
     @validates_schema
     def validate_schema_name(self, data, **kwargs):
         # pylint: disable=no-self-use, unused-argument
-        """Temporary function for census to validate the census schema parameters
-        This can be removed after census.
-        """
-        individual_schema_claims = (
-            data.get("survey"),
+        """Function to validate the business schema parameters"""
+        business_schema_claims = (
```

Comment:
Can this not be moved down to after the if?

Reply:
Done, thanks!